docs/how-to-guides/integrating-autoware/launch-autoware/localization-mode/index.md

```diff
@@ -0,0 +1,85 @@
+# Localization mode
+
+The Autoware provides multiple localization methods that work with multiple different sensor configurations.
+The table below shows the supported sensor configurations and their corresponding algorithms.
+
+| localization mode | method  | map type        |
+| ----------------- | ------- | --------------- |
+| LiDAR-based       | NDT     | point cloud map |
+| camera-based      | YabLoc  | vector map      |
+| GNSS/IMU-based    | Eagleye | -               |
+
+## LiDAR-based localizer (default)
+
+By default, Autoware launches [ndt_scan_matcher](https://github.com/autowarefoundation/autoware.universe/tree/main/localization/ndt_scan_matcher) for localization.
+
+## Camera-based localizer
+
+You can use YabLoc as a camera-based localization method. For more details on YabLoc, please refer to the [YabLoc Guide](yabloc-guide.md).
+
+To use YabLoc as a pose_estimator, add `localization_mode:=camera` when launching Autoware.
+By default, the `localization_mode` is set to `lidar`.
+By specifying this command-line argument, YabLoc nodes will be automatically launched while the NDT nodes will not be started.
+
+Here is an example of a launch command:
+
+```bash
+ros2 launch autoware_launch autoware.launch.xml \
+  vehicle_model:=YOUR_VEHICLE \
+  sensor_kit:=YOUR_SENSOR_KIT map_path:=/PATH/TO/YOUR/MAP \
+  localization_mode:=camera # Add this argument
+```
+
+## GNSS/IMU-based localizer
+
+You can use Eagleye as a GNSS/IMU-based localization method. For more details on Eagleye, please refer to the [Eagleye Guide](eagleye-guide.md).
+
+Eagleye has a function for position estimation and twist estimation, namely `pose_estimator` and `twist_estimator`, respectively.
+When running Eagleye in twist_estimator mode, ndt_scan_matcher is used as the pose_estimator.
+Eagleye will improve scan matching by providing accurate twists using GNSS doppler.
+
+To use Eagleye, it requires both specifying the command-line arguments and modifying the launch file.
+
+### 1. Modifying Autoware launch files
```

Comment:
It's just a nits so can be ignored.

IMHO it's a bit confusing to have subsubsections only for Eagleye in terms of TOC (see the figure below). Given that these two subsubsections are not that long, how about just removing these titles?
```suggestion
```

![image](https://github.com/autowarefoundation/autoware-documentation/assets/44218668/7bced061-dab5-4478-b256-cc2d0c22c1b6)

Reply:
I merged the short sub-sub-sections into sub-sections in 50ff60b